istioctl/pkg/writer/envoy/configdump/listener.go

```diff
@@ -104,7 +104,17 @@ func retrieveListenerType(l *listener.Listener) string {
 }
 
 func retrieveListenerAddress(l *listener.Listener) string {
-	return l.Address.GetSocketAddress().Address
+	sockAddr := l.Address.GetSocketAddress()
+	if nil != sockAddr {
```

Comment:
```suggestion
	if sockAddr != nil {
```

Reply:
thanks for your comments, I have improve the coding style.